README.md

```diff
@@ -66,10 +66,9 @@ Don't forget to give the project a star! Thanks again!
 1. Fork the Project
 2. Create your Feature Branch (`git checkout -b feature/cool-feature`)
 3. Implement your desired Feature!
-3. Commit your Changes (`git commit -am 'Add some cool-feature'`)
-4. Push to the Branch (`git push origin feature/cool-feature`)
-5. Create a new [Pull Request](https://github.com/benzammour/yobemag/pulls)
-
+4. Commit your Changes (`git commit -am 'Add some cool-feature'`)
```

Comment:
`git commit -a` can be a bit dangerous if one is working with IDEs, that none of us use.

Reply:
What alternative do you suggest? 🤔